### PR TITLE
Fix failing CI

### LINKED ISSUE #29: CI is broken
[As mentioned](https://github.com/epitools/napari-epitools/pull/28#issuecomment-1453448837) in a previous pr, tests pass locally but CI is failing, possibly due to an issues with `pytest-xvfb`

### PR DESCRIPTION
**Pull request by p-j-smith**
Fixes #29

### CODE REVIEW

**Comment by p-j-smith** [comment]
okay it's working now but there might be a better way to fix the issue, see https://github.com/epitools/napari-epitools/issues/29#issuecomment-1453706795 for details

File: .github/workflows/test_and_deploy.yml [hunk 1]
```diff
@@ -32,14 +33,7 @@ jobs:
...
 
       # these libraries enable testing on Qt on linux
-      - uses: tlambert03/setup-qt-libs@v1
-
-      # strategy borrowed from vispy for installing opengl libs on windows
-      - name: Install Windows OpenGL
-        if: runner.os == 'Windows'
-        run: |
-          git clone --depth 1 https://github.com/pyvista/gl-ci-helpers.git
-          powershell gl-ci-helpers/appveyor/install_opengl.ps1
```
Comment (paddyroddy) [inline .github/workflows/test_and_deploy.yml:42]: Do we need/want this Windows thing?

**Review by paddyroddy — APPROVED** [review]
Nice work

File: .github/workflows/test_and_deploy.yml (hunk 1 above)
Comment (p-j-smith) [reply .github/workflows/test_and_deploy.yml:42]: I was just updating the workflow to match the [current version](https://github.com/napari/cookiecutter-napari-plugin/blob/main/%7B%7Bcookiecutter.plugin_name%7D%7D/.github/workflows/test_and_deploy.yml) in current napari-plugin. But I think you're right that it's not necessary - I'll remove it

Comment (p-j-smith) [reply .github/workflows/test_and_deploy.yml:42]: ah sorry I have already removed it! Do you want me to add it back?

Comment (paddyroddy) [reply .github/workflows/test_and_deploy.yml:42]: If it's not needed, happy for it to go

**Pull request merged**